external-control/src/main/cpp/controllers/constant-position-controller.cpp

```diff
@@ -0,0 +1,61 @@
+#include "constant-position-controller.hpp"
+
+#include "types.hpp"
+
+namespace ihmc
+{
+    ConstantPositionController::ConstantPositionController(const double default_stiffness, const double default_damping, const int number_of_joints)
+    {
+        number_of_joints_ = number_of_joints;
+        desired_joint_velocities_ = Eigen::VectorXd(number_of_joints_);
+        desired_joint_torques_ = Eigen::VectorXd(number_of_joints_);
+        desired_joint_stiffnesses_ = Eigen::VectorXd(number_of_joints_);
+        desired_joint_damping_ = Eigen::VectorXd(number_of_joints_);
```

Comment:
use .resize(Number_of_size); instead using Eigen::VectorXd(#). This may cause the memory issue.